prdoc/pr_7043.prdoc

```diff
@@ -0,0 +1,21 @@
+title: Remove usage of `sp-std` from Substrate
+author: conr2d
+topic: runtime
+
+crates:
+  - name: frame-support
+    bump: none
```

Comment:
I am not sure the bump is none, because the change is needed by frame-support-procedural to work.

Reply:
Yes, but if somebody make a change to `frame-support-procedural` and bump as a patch and release, and in the meantime frame-support doesn't have any other change, then the new patch version of `frame-support-procedural` is broken.
All in all I think it will be fine, but maybe doing a minor or patch bump for both `frame-support` and `frame-support-procedural` is more correct.

Reply:
Yeah let's make all of them at least `patch`.

Reply:
> Yes, but if somebody make a change to `frame-support-procedural` and bump as a patch and release, and in the meantime frame-support doesn't have any other change, then the new patch version of `frame-support-procedural` is broken.

procedural doesn't require any changes in support, but as I said just make all of them `patch`.